gitenberg/config.py

```diff
@@ -67,6 +67,7 @@ def read(self):
     def parse(self):
         global data
         data = yaml.load(self.read())
+        self.data = data
```

Comment:
Here I've added `config.data` and new classes I create can use this one.  Eventually I'd like to use the Singleton pattern to do this as a global 'properly' but not today